i18n/ja/docusaurus-plugin-content-docs/current/develop/guides/cheat-sheets/momento-cache-python-cheat-sheet.md

```diff
@@ -128,15 +128,15 @@ An example of incrementing a value in a key. You can pass in a positive or negat
 ```
 
 :::info
-Beyond these basic API calls check out the [API reference page](../../api-reference/index.mdx) for more information on the full assortment of Momento API calls.
+これらの基礎的な API 呼出しに加えて、Momento API 呼出しの全ての一覧がある [API リファレンスページ](../../api-reference/index.mdx)もご覧下さい。
 :::
 
-## Notes of further usage
+## さらなる利用に向けて
 
-It could be adventageous to reuse the CacheClient connection object across multiple API calls depending on how your app works. If this is something you want to do, then you should modify the client() function definition to return a CacheClient object like so:
+アプリケーションの挙動によっては、複数の API 呼出しの間で CacheClient オブジェクトを再利用した方が良い場合があるでしょう。その様な場合には、 client() 関数の定義を以下の様に変えて、CacheClient オブジェクトを返す様にすると良いでしょう:
```

Comment:
BTW, I'm not sure about what this paragraph means since `client()` already looks like returning `CacheClient` object.

Reply:
@riywo 
[Here](https://github.com/momentohq/public-dev-docs/pull/197/files#diff-748a493d3a38089ae24cab0f541d22ffcab70a43299850aa5e4bc44ee58fac1cR30-R39), `def client()` in `MomentoBasics` class is instantiating a `CacheClient` object. 
So every time you call `MomentoCounter.client()` a new `CacheClient` object is created in these operations in the examples such as [creating a cache](https://github.com/momentohq/public-dev-docs/pull/197/files#diff-748a493d3a38089ae24cab0f541d22ffcab70a43299850aa5e4bc44ee58fac1cR48-R56) and [listing caches](https://github.com/momentohq/public-dev-docs/pull/197/files#diff-748a493d3a38089ae24cab0f541d22ffcab70a43299850aa5e4bc44ee58fac1cR64-R76).

So by changing `def client()` to return just an object `CacheClient` not an instantiated object and pass this object type to each operations I mentioned above's as their arguments, then you can pass one instantiated `CacheClient` object instead of creating a `CacheClient` object every API call.

Reply:
But this is good feedback. 
If it doesn't make sense after reading this page, we should update it so that it's clear what we're trying to communicate.

Reply:
You probably just need `@cache` ? https://docs.python.org/3/library/functools.html#functools.cache

Reply:
thank you for the feedback @riywo !
It looks like the cheat sheet for the Python SDK is not clear enough to convert our intention to users.
We'll discuss internally and update this page accordingly.